README.md

```diff
@@ -1,227 +1,450 @@
 
+  - **Choose an image**: Distribution -> Ubuntu -> Version: 22.04 (LTS) x64
+  - **Choose a plan**: 
+    - Shared CPU: Basic
+    - CPU Options: Regular
+    - Price: $6/mo (Specs: 1 GB/1 CPU, 25 GB SSD Disk, 1000 GB Transfer)
+  - **Add block storage**: Skip this step
+  - **Choose a datacenter region**: Toronto 1
+  - **VPC Network**: Skip this step
+  - **Authentication**: If your key isn't present yet, press "New SSH Key" and follow the instructions to authenticate the device you're currently using.
+  - **Select additional options**: Check "Monitoring" and leave the rest blank.
+  - **How many Droplets?**: 1 Droplet
+  - **Choose a hostname**: This is a very critical step! Make sure you enter the same domain name you chose earlier (`{subdomain}.spinalcordmri.org`)
+  - **Add tags**: Skip this step
+
+Once the droplet has finished creating, you should see a public IP address on the droplet's page that looks something like `ipv4: 142.93.152.255`. You can then use this IP address to test that you've set the hostname correctly by running the following command on any linux machine:
+
+```console
+user@device:~$ dig +short -x 142.93.152.255
+{subdomain}.spinalcordmri.org.
 ```
-root@forum:~# ls -l /var/discourse/shared/standalone/ssl/forum.spinalcordmri.org.{cer,key}
--rw-r--r-- 1 root root 3799 Dec  5 08:33 /var/discourse/shared/standalone/ssl/forum.spinalcordmri.org.cer
--rw------- 1 root root 3247 Dec  5 08:33 /var/discourse/shared/standalone/ssl/forum.spinalcordmri.org.key
+
+Once you've confirmed that it returns the domain name you chose, you're all set to connect to the droplet.
+
+----
+
+### 2. Namecheap
+
+Before we make any changes to the server, though, we first have to configure some DNS settings.
+
+First, make sure you have an account with Namecheap. Again, you will need to contact someone on the admin team; they will grant you permissions for specific domains on a case-by-case basis. In this case, you will want access to the spinalcordmri.org domain, which will grant you access to the [Spinalcordmri.org Dashboard](https://ap.www.namecheap.com/domains/domaincontrolpanel/spinalcordmri.org/domain).
+
+Next, on the dashboard, navigate to the [Advanced DNS tab](https://ap.www.namecheap.com/Domains/DomainControlPanel/spinalcordmri.org/advancedns). Then, modify the following sections:
+
+#### 2.1 Host Records
+
+First you will want to make note of two things:
+
+1. The subdomain label of your domain name. For `{subdomain}.spinalcordmri.org`, you will use `{subdomain}` to identify your subdomain.
+2. The IP address of the server (from step 1). For `{subdomain}.spinalcordmri.org`, the IP address was `142.93.152.255`.
+
+You can now use the red "Add New Record" button to add 3 new records in the following form:
+
+Type | Host | Value | TTL
+-- | -- | -- | --
+A Record | {subdomain} | 142.93.152.255 | Automatic
+TXT Record | {subdomain} | v=spf1 a mx ip4:142.93.152.255 include:spf.efwd.registrar-servers.com ~all | Automatic
```

Comment:
I'm 99% sure spf.efwd.registrar-servers.com is NameCheap's email forwarding service but I'm not clear. Registrars always have really abstract names that don't seem to match up. It's a bit sketch to let something we're not using and not sure about have the rights to send email in our name.

Reply:
Thanks for pointing this out! I definitely copied and pasted from the existing config, so I'm glad you interrogated that a bit further.

Reply:
> The thing is, we don't actually use this email forwarding service, do we?

Indeed!

![image](https://user-images.githubusercontent.com/16181459/211342432-5221ff64-86c1-459d-8a1f-e4b899b1c0e8.png)